projects/js-packages/number-formatters/package.json

```diff
@@ -0,0 +1,38 @@
+{
+	"name": "@automattic/jetpack-number-formatters",
```

Comment:
You're welcome to rename this to something like `@automattic/number-formatters` if you want. Including "jetpack-" isn't required. 🙂